packages/fx-core/tests/common/deps-checker/cases/node.it.ts

```diff
@@ -8,7 +8,6 @@ import { TestTelemetry } from "../adapters/testTelemetry";
 import { DepsType } from "../../../../src/common/deps-checker/depsChecker";
 import { CheckerFactory } from "../../../../src/common/deps-checker/checkerFactory";
 import "mocha";
-import { func } from "ajv/dist/compile/util";
```

Comment:
Just for quick PR process... may fix this in another PR to avoid the long running integration test...

Reply:
Anyway, not required.